members/B000944.yaml

```diff
@@ -50,7 +50,7 @@ contact_form:
       - name: accent-replace
         values: ["$NAME_FIRST", "$NAME_LAST"]
         selectors: ["#input-466BAE61-E646-3957-206E-507B30B9CC9C", "#input-466BB01B-E6B3-9F1C-54B0-74E30CA82B2D"]
-        commands: ["for(var i = 0; i < elements.length; i++) { elements[i].value = values[i].replace(/['á']/g, 'a').replace(/[èé]/g,'e').replace(/[ñ]/g, 'n').replace(/[']/g, ''); }"]
+        commands: ["for(var i = 0; i < elements.length; i++) { elements[i].value = values[i].replace(/['á']/g, 'a').replace(/[èé]/g,'e').replace(/[ñ]/g, 'n').replace(/[']/g, '').replace(/['%20']/g, ''); }"]
```

Comment:
You want: `.replace(/(%20)/g, '')`
Putting something within `[ ]` in regex defines a character set, rather than a pattern to match

Reply:
chatted with Tommy 1-on-1. Regex in the js step has been updated to `replace(/(%2[0C])/g, '')`